networks/remote/README.md

```diff
@@ -1,3 +1,3 @@
 # Remote Cluster with Terraform and Ansible
 
-See the [docs](/docs/terraform-and-ansible.md)
+See the [docs](https://tendermint.com/docs/networks/terraform-and-ansible.html).
```

Comment:
don't we want Github links to still work?

Reply:
Like we should just link to the github markdown here instead of the website docs?

Reply:
Yes

Reply:
/cc @zramsay

Reply:
see #links in the `DOCS_README`

Reply:
should be relative links - they work everywhere and are permanent